### ISSUE TITLE
Add bindings for rust

### ISSUE THREAD

**Issue by flukejones** [jrmoulton/tree-sitter-slint#9]
Thanks for doing this work. I was about to add this to lapce but there are missing bindings, can you add them please?

Something like https://github.com/UserNobody14/tree-sitter-dart/tree/master/bindings

**Comment by jrmoulton** [comment]
I intentionally left them out because they are just automatically generated by the tree-sitter cli and I didn't want to check them in to the repo. I'd still rather not check them in. Does Lapce need them to be in the repo?

**Comment by flukejones** [comment]
Yeah it does. Seems like helix-editor also requires it.

**Comment by jrmoulton** [comment]
Helix supports slint highlighting and didn't require the bindings

**Comment by jrmoulton** [comment]
But I can add them if it's needed. It's not the end of the world I guess

**Comment by flukejones** [comment]
> Helix supports slint highlighting and didn't require the bindings

Oh I didn't realise it was already supported. That's cool!

**Comment by flukejones** [comment]
> But I can add them if it's needed. It's not the end of the world I guess

Maybe as a temporary thing? I'll raise an issue with lapce.

**Comment by flukejones** [comment]
Hey boss, looks like every other tree-sitter out there including all the ones in the official repo are including the generated bindings in the repo (does not build with cargo unless included).

**Comment by flukejones** [comment]
Add PR for lapce to use this - https://github.com/lapce/lapce/pull/2407

Will submit a new PR to change remote for it once my PR here is merged.
